lib/utils/udpsender.js

```diff
@@ -0,0 +1,20 @@
+require('./KalturaConfig');
+
+var PORT = KalturaConfig.config.udpsender.PORT;
+var HOST = KalturaConfig.config.udpsender.HOST;
+var client = require('dgram').createSocket('udp4');
+
+var sendFunction = function (message,response){
+    var message_buffer = new Buffer(message);
+    response.log('#UDP ' + message_buffer + ' sent to ' + HOST + ':' + PORT);
+    client.send(message_buffer, 0, message_buffer.length,PORT, HOST,
+        function (err, bytes) {
+            if (err) {
+                client.close();
+            }

```

Comment:
@etameran - I agree UDP is light wight - however the motivation for this change is not to receive real-times alerts but to be able to answer beacon/fill-rate differences questions faster and in more convenient way.  No need for real-time at all. Also this depends on how well the UDP client works - if significant (yet small in absolute numbers) number of logs will fail to deliver - we'll end up looking into raw logs again to make sure the numbers we provide are accurate. 

Another concern is that errors/exceptions of this sender will interfere with the normal process behavior - so the erroneous cases should be tested,

Last there's no need to parse anything, even if using external process: we can write the log messages in JSON format, grep for these special logs only and send them over (UDP or any other method)